float32 left_measured_angle_rad
float32 left_estimated_angular_velocity